definitions/procedures/sync_plans/enable.rb

```diff
@@ -5,6 +5,10 @@ class Enable < ForemanMaintain::Procedure
       description 're-enable sync plans'
       tags :post_migrations
       before :disk_io
+
+      confine do
+        feature(:katello)
```

Comment:
shouldnt a pulp feature be here? On capsule, there is no katello, but there is pulp

Reply:
Sure Ivan, I am trying to find the package we need to check for `Pulp` feature.
I suppose it should be pulp-server.
Thoughts?
```
sabnave-satellite-1 :: ~ » rpm -qa "*pulp*"   
pulp-client-1.0-1.noarch
pulp-docker-plugins-2.0.3-1.el7sat.noarch
pulp-server-2.8.7.3-1.el7sat.noarch
python-pulp-common-2.8.7.3-1.el7sat.noarch
python-pulp-repoauth-2.8.7.3-1.el7sat.noarch
pulp-puppet-tools-2.8.7.1-1.el7sat.noarch
python-pulp-rpm-common-2.8.7.5-1.el7sat.noarch
pulp-puppet-plugins-2.8.7.1-1.el7sat.noarch
rubygem-smart_proxy_pulp-1.2.2-1.el7sat.noarch
python-pulp-oid_validation-2.8.7.3-1.el7sat.noarch
pulp-katello-1.0.1-1.el7sat.noarch
python-pulp-bindings-2.8.7.3-1.el7sat.noarch
python-pulp-puppet-common-2.8.7.1-1.el7sat.noarch
pulp-selinux-2.8.7.3-1.el7sat.noarch
pulp-rpm-plugins-2.8.7.5-1.el7sat.noarch
python-pulp-docker-common-2.0.3-1.el7sat.noarch
python-pulp-streamer-2.8.7.3-1.el7sat.noarch
```